docs-content/release-notes.html.md.erb

```diff
@@ -5,6 +5,48 @@ owner: Partners
 
 These are release notes for CyberArk Conjur Service Broker for VMware Tanzu
 
+## <a id="ver"></a> v1.3.0
+
+**Release Date:** August 9, 2021
+
+- CyberArk Conjur Service Broker - [v1.2.1](https://github.com/cyberark/conjur-service-broker/releases/v1.2.1)
+- CyberArk Conjur Buildpack - [v2.2.1](https://github.com/cyberark/cloudfoundry-conjur-buildpack/releases/v2.2.1)
+
+Features in this release:
+
+* Removed support for Conjur Enterprise v4
+  - We recommend migrating to Dynamic Access Provider v11+ or Conjur OSS v1+.
```

Comment:
Should references to DAP be changed to Conjur Enterprise?

Reply:
Hmm yeah I guess it should. I thought we had to leave this one in for some reason but I don't think that's the case. Will fix.